appserver/common/container-common/src/main/java/com/sun/enterprise/container/common/impl/managedbean/ManagedBeanManagerImpl.java

```diff
@@ -659,15 +631,12 @@ public void destroyManagedBean(Object managedBean, boolean validate)  {
                 Field proxyField = managedBean.getClass().getDeclaredField("__ejb31_delegate");
 
                 final Field finalF = proxyField;
-                    java.security.AccessController.doPrivileged(
-                    new java.security.PrivilegedExceptionAction() {
-                        public java.lang.Object run() throws Exception {
-                            if( !finalF.isAccessible() ) {
-                                finalF.setAccessible(true);
-                            }
-                            return null;
-                        }
-                    });
+                java.security.AccessController.doPrivileged((PrivilegedExceptionAction<Object>) () -> {
```

Comment:
If you would also extract the action to the variable, you don't need casting it.